sections/main-password-header.liquid

```diff
@@ -9,7 +9,7 @@
     {%- if section.settings.logo != blank -%}
       {%- assign logo_alt = section.settings.logo.alt | default: shop.name | escape -%}
       {%- assign logo_height = section.settings.logo_max_width | divided_by: section.settings.logo.aspect_ratio -%}
-      {{ section.settings.logo | image_url: width: section.settings.logo.width | image_tag:
+      {{ section.settings.logo | image_url: width: 500 | image_tag:
```

Comment:
Here it's a bit different, since the logo uploaded could be much wider, we don't really need to have something that big. So I default to the max value it's going to use instead.

Reply:
I think here is the same thing that I mentioned in the first comment about logo in header.liquid. I don't see what changed. It seems even with previous setting `width: section.settings.logo.width` it doesn't go higher than a biggest value from  `widths: '50, 100, 150, 200, 250, 300, 400, 500',` (line 14).

Reply:
This is to prevent anything too big from being loaded. If we know the max value we're outputting is `500` no need to load an image bigger than that.